src/components/admin/MainIssueController.jsx

```diff
@@ -156,8 +156,27 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
           }
+          if (/<a.*?href\s*?=\s*?["'](?!http).*?>/.test(article.html)) {
```

Comment:
Haha, you're somehow merging this in again, let's not do that :)

P.S: Just for clarity, that means undo all the changes in this file